configs/default/VGRC-VGOODRCF4.config

```diff
@@ -1,38 +1,69 @@
-# Betaflight / STM32F405 (S405) 4.1.0 Apr 30 2019 / 09:30:28 () MSP API: 1.42
+# Betaflight / VGOODRCF4 (VGF4) 4.0.2 May 30 2019 / 10:19:58 () MSP API: 1.41
```

Comment:
There are a number of problems with these changes - please read the instructions in https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md and follow them closely to fix the problems.

Reply:
Thanks for your remind. We don't sell  products with old version to the market, as when we test, we find it need to improve, so please replace with this new, thanks.